src/providers/search/search.ts

```diff
@@ -11,16 +12,61 @@ export class SearchProvider {
     public urlProvider : UrlProvider,
   ) {}
 
-  driverNumberSearch(driverNumber: string): Observable<any> {
+  driverNumberSearch(driverNumber: string, isLDTM: boolean = false): Observable<any> {
     return this.http.get(
-      this.urlProvider.getTestResultServiceUrl().concat(`?driverNumber=${driverNumber}`),
+      this.urlProvider.getTestResultServiceUrl()
+        .concat(`?driverNumber=${driverNumber}&isLDTM=${isLDTM}`),
     );
   }
 
-  applicationReferenceSearch(applicationReference: String): Observable<any> {
+  applicationReferenceSearch(applicationReference: string, isLDTM : boolean = false): Observable<any> {
     return this.http.get(
-      this.urlProvider.getTestResultServiceUrl().concat(`?applicationReference=${applicationReference}`),
+      this.urlProvider.getTestResultServiceUrl()
+        .concat(`?applicationReference=${applicationReference}&isLDTM=${isLDTM}`),
     );
   }
 
+  advancedSearch(advancedSearchParams: AdvancedSearchParams): Observable<any> {
+
+    const httpParams: string [] = [];
+
+    advancedSearchParams.isLDTM ?
+     httpParams.push('isLDTM=true') : httpParams.push('isLDTM=false');
```

Comment:
quick todo comment to remove this when https://jira.i-env.net/browse/MES-2135 is implemented

Reply:
done